src/components/Home/HomeSection.jsx

```diff
@@ -46,10 +46,11 @@ const HomeSection = ({
       mb="24px"
       sx={{
         color: 'primary.main',
-        fontSize: '28px'
+        fontSize: '28px',
+        fontWeight: 'bold'
       }}
     >
-      <strong>{title}</strong>
```

Comment:
With regards to semantic elements, think it'll be better to lean more on native elements than MUI components whenever we can. But that's just my opinion on this. Don't need to revert if you think it's fine.

Reply:
That's fine with me. I was going off of how it's done elsewhere, since there are already multiple instances of using `fontWeight` and only a couple of `<strong>`s.

Which means we should set up a definitive standard and do a full review sometime.